wal/encoder.go

```diff
@@ -36,7 +36,7 @@ type encoder struct {
 
 func newEncoder(w io.Writer, prevCrc uint32) *encoder {
 	return &encoder{
-		bw:  bufio.NewWriter(w),
+		bw:  ioutil.NewPageWriter(w, 4096),
```

Comment:
@gyuho it's only a coincidence the wal page size is the bufio default buffer size; the full buffer size for a pagewriter is 128k. I'll add a comment about the choice of the value.